synapseutils/sync.py

```diff
@@ -770,6 +771,18 @@ def readManifestFile(syn, manifestFile):
         raise ValueError("All rows in manifest must contain a unique file to upload")
     sys.stdout.write('OK\n')
 
+    # Check each size of uploaded file
+    sys.stdout.write('Validating that all the files are not empty...')
+    _check_size_each_file(df)
+    sys.stdout.write('OK\n')
+
+    # check the name of each file should be store on Synapse
+    name_column = 'name'
+    if name_column in df.columns:
+        sys.stdout.write('Validating that all name column are filled... \n')
```

Comment:
Maybe "Validating file names..." since it is doing more than just checking that the names are filled (checking length, characters etc).